ui/components/app/import-token-link/import-token-link.component.js

```diff
@@ -37,8 +42,15 @@ export default function ImportTokenLink() {
           addTokenEvent();
         }}
       >
-        {t('importTokens')}
+        {isMainnet
+          ? t('importTokens')
```

Comment:
Any reason we can't just use `t('importTokensCamelCase')` here for non-mainnet networks??

Reply:
The message in `t('importTokensCamelCase')` is `Import Token` but we need `Import token` here

Reply:
or if need be can we add a new translation called `'importTokensLabel'` that equals `"Import tokens"`? I now see it's for consistency with `refreshList`

Reply:
Yes, but I was not sure if it is worth adding multiple entries for the same message just for case change, if I add this one that will be 3rd entry for the same message.